cmd/rigs/cmd/publish_data.go

```diff
@@ -181,28 +222,68 @@ func attrsJobs(ctx context.Context, s local.Store, jobID *int) ([]wpool.Job, err
 	return jobs, nil
 }
 
+func dealsJobs(ctx context.Context, s local.Store, jobID *int) ([]wpool.Job, error) {
+	var jobs []wpool.Job
+	var offset uint
+	dealsExecFn := func(rigs []local.Rig) wpool.ExecutionFn {
+		return func(ctx context.Context) (interface{}, error) {
+			if err := store.InsertDeals(ctx, rigs); err != nil {
+				return nil, fmt.Errorf("calling insert deals: %v", err)
+			}
+			return nil, nil
+		}
+	}
+	for {
+		rigs, err := s.Rigs(ctx, local.RigsWithLimit(dealsPageSize), local.RigsWithOffset(offset))
+		if err != nil {
+			return nil, fmt.Errorf("getting rigs: %v", err)
+		}
+		if len(rigs) == 0 {
+			break
+		}
+
+		jobs = append(
+			jobs,
+			wpool.Job{
+				ID:     wpool.JobID(*jobID),
+				ExecFn: dealsExecFn(rigs),
+				Desc:   fmt.Sprintf("deals offset %d", offset),
+			},
+		)
+		*jobID++
+		offset += dealsPageSize
+	}
+	return jobs, nil
+}
+
 func lookupsJob(s local.Store, jobID *int) wpool.Job {
 	return wpool.Job{
 		ID: wpool.JobID(*jobID),
 		ExecFn: func(ctx context.Context) (interface{}, error) {
 			rendersCid, err := s.Cid(ctx, "renders")
 			if err != nil {
-				return nil, fmt.Errorf("querying images cid: %v", err)
+				return nil, fmt.Errorf("querying renders cid: %v", err)
 			}
 			layersCid, err := s.Cid(ctx, "layers")
 			if err != nil {
 				return nil, fmt.Errorf("querying layers cid: %v", err)
 			}
+			indexCid, err := s.Cid(ctx, "index")
+			if err != nil {
+				return nil, fmt.Errorf("querying index cid: %v", err)
+			}
 			lookups := tableland.Lookups{
 				RendersCid:           rendersCid,
 				LayersCid:            layersCid,
+				IndexCid:             indexCid,
 				ImageFullName:        "image_full.png",
 				ImageFullAlphaName:   "image_full_alpha.png",
 				ImageMediumName:      "image_medium.png",
 				ImageMediumAlphaName: "image_medium_alpha.png",
 				ImageThumbName:       "image_thumb.png",
 				ImageThumbAlphaName:  "image_thumb_alpha.png",
 				AnimationBaseURL:     "https://rigs.tableland.xyz/",
+				FilecoinBaseURL:      "https://filfox.info/deal/",
```

Comment:
This is the url that we end up using in the metadata query. Are we cool with pointing to Filfox or should we use Filescan or other?